index.go

```diff
@@ -111,6 +111,14 @@ func (i *IndexClient) DeleteIndex(ctx context.Context, index string) (*elastic.I
 	return i.raw.DeleteIndex(index).Do(ctx)
 }
 
+func (i *IndexClient) Delete(ctx context.Context, index string, id string) (*elastic.DeleteResponse, error) {
```

Comment:
You could write `fucn (i *IndexClient) Delete(ctx context.Context, index, id string)` for short.

Reply:
Thank for your comment.
I fixed it,